library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942_80cu/Equality/aquavanjaram_Cijk_Alik_Bljk_HHS_BH_UserArgs.yaml

```diff
@@ -860,13 +860,277 @@
     _WorkspaceSizePerElemBias: 0
     _WorkspaceSizePerElemC: 4
     _staggerStrideShift: 0
+  - 1LDSBuffer: 1
+    ActivationAlt: false
+    ActivationFuncCall: false
+    ActivationFused: true
+    AssertFree0ElementMultiple: 16
+    AssertFree1ElementMultiple: 1
```

Comment:
Does this really work with N not multiple of 16?

Reply:
tested with N=1 to 16, working with no issues.